Makefile

```diff
@@ -92,3 +92,32 @@ delete-awsfederatedaccountaccess: check-aws-account-id-env
 
 .PHONY: test-awsfederatedaccountaccess
 test-awsfederatedaccountaccess: check-aws-account-id-env create-account create-awsfederatedrole create-awsfederatedaccountaccess test-switch-role delete-awsfederatedaccountaccess delete-awsfederatedrole delete-account
+
+.PHONY: deploy-aws-account-operator-credentials
+deploy-aws-account-operator-credentials:
+# Base64 Encode the AWS Credentials
+	$(eval ID=$(shell base64 <<< ${AWS_ACCESS_KEY_ID}))
```

Comment:
Are you assuming here that people have their `.aws/credentials` setup without profiles?

Reply:
No, the ID and Key are needed as parameters for the credentials secret created from the `aws_v1alpha1_aws_account_operator_credentials.yaml` further down.  I was following the precedent of defining a variable as done for the check-aws-account-id-env target, for consistency.

A deploy would look like: `AWS_ACCESS_KEY_ID="something" AWS_SECRET_KEY_ID="something else" make deploy`

This was also an easy way to base64 encode the credentials before using them in the secret template.

Reply:
I think it would be nice if it assumed `.aws/credentials` was setup. I don't know if this is possible but being able to apply secrets interchangeably between osd-1 & osd-2 would be nice as well.

Reply:
@ArielLima: It's not an AWS limitation, though, otherwise I would just use the `.aws/credentials` file directly as you and James have suggested.

As I mentioned above, the Key and ID are required as parameters to create the account secret in MiniShift, so they have to be parsed and used by the Makefile script.  They can't just be left up to the `aws` command to handle.

Reply:
It would be nice to not have to put in the secrets if they're already in the credentials file, but I don't think that should hold this up.  We should add a backlog item to figure out a way to get those out of the credentials file.

Reply:
Going with @jharrington22's request, I've replaced the use of `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` with `OPERATOR_ACCESS_KEY_ID` and `OPERATOR_SECRET_ACCESS_KEY` to avoid conflict with AWS CLI tool creds.

Parsing an `.aws/credentials` file is impractical as it requires mandating the format of everyone's creds file, which I'd like to avoid.